src/editors/data/redux/thunkActions/app.js

```diff
@@ -103,12 +103,8 @@ export const initialize = (data) => (dispatch) => {
       dispatch(module.fetchCourseDetails());
       break;
     case 'html':
-      if (isLibraryKey(data.learningContextId)) {
-        // eslint-disable-next-line no-console
-        console.log('Not fetching image assets - not implemented yet for content libraries.');
-      } else {
-        dispatch(module.fetchImages({ pageNumber: 0 }));
-      }
+      if (isLibraryKey(data.learningContextId)) { dispatch(actions.app.resetImages()); }
```

Comment:
Why do we need to `resetImages()` here, any why only for libraries?

Reply:
When I was testing the functionality I notice that closing the modal don't clean the state, so I can see the images of one component in the other one. 

My assumption about why we can see that behavior in courses is not happening in courses because we have a change in the URL (from the legacy unit view to the MFE) that clean the state of the component.